spring-cloud-gcp-data-spanner/pom.xml

```diff
@@ -34,5 +34,33 @@
 			<groupId>org.springframework</groupId>
 			<artifactId>spring-core</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-gcp-core</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-tx</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-gcp-autoconfigure</artifactId>
```

Comment:
Yep, this should probably be left out completely. Just add a separate configuration providing all your test `@Bean`s. It might feel like duplication, but I think this is expected for tests. Even for user configuration, when Spring Boot isn't used.

Reply:
Oh! Wait. I thought all these tests are in the `autoconfigure` module.

If that is otherwise, and there is not Boot dependency in this `spring-cloud-gcp-data-spanner` module, then yes: we don't have choice unless to bring some copy/paste hell for us with the configuration, but make these tests to work.

Reply:
Alright. For now I'll go this direction and duplicate it to the integration tests. But it feels like this might come back as a user request to have the ability to configure the lib without spring boot. There are definitely parts of the configuration that are never supposed to change - hence could be safely part of the library, instead of forcing the user to build out everything from scratch...